File: src/26-challenge/solution.py
from datetime import datetime


def get_completed(time_worked: str, total_time: str) -> str:
    worked: datetime = datetime.fromisoformat(f"1970-01-01T${time_worked}")
    total_worked: datetime = datetime.fromisoformat(f"1970-01-01T${total_time}")
    productivity: str = (
        f"${round(number=(worked.timestamp() / total_worked.timestamp()) * 100)}%"
    )
    return productivity

File: src/26-challenge/solution.js
/**
 * @param {string} timeWorked - Time worked in hh:mm:ss format.
 * @param {string} totalTime - Total estimated time in hh:mm:ss format.
 * @returns {string} - The completed percentage rounded to the nearest integer with a % sign.
 */
export default function getCompleted(timeWorked, totalTime) {
	/** @type {Date} */
	const worked = new Date(`1970-01-01T${timeWorked}`)

	/** @type {Date} */
	const totalWorked = new Date(`1970-01-01T${totalTime}`)

	/** @type {string} */
	const productivity = `${Math.round((worked.getTime() / totalWorked.getTime()) * 100)}%`

	return productivity
}
